tests/accessor/accessor_default_values_fp16.cpp

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for accessor default values.
+//
+//  This test provides verifications that template parameters has default values
+//  for generic accessor, host_accessor and local_accessor with sycl::half type.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "../common/disabled_for_test_case.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+
+#include "accessor_default_values.h"
+
+using namespace accessor_default_values_test;
+using namespace accessor_tests_common;
+#endif
+
+namespace accessor_exceptions_test_fp16 {
+using namespace sycl_cts;
+
+DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
+("Accessors constructor default values test fp16 types.", "[accessor]")({
+#ifndef SYCL_CTS_ENABLE_FULL_CONFORMANCE
+  run_tests<sycl::half>{}("sycl::half");
+#else
+  for_type_vectors_marray<run_tests, sycl::half>("sycl::half");
+#endif  // SYCL_CTS_ENABLE_FULL_CONFORMANCE
```

Comment:
I think 
```c++
#ifdef A
  X
#else
  Y
#endif
```
is more readable than
```c++
#ifndef A
  Y
#else
  X
#endif
```
Please, prefer the version w/o negation in condition to reduce mental load of the code.